pages/api/enroll/child.ts

```diff
@@ -163,6 +167,23 @@ export default async function handle(
                 return;
             }
 
+            // check if the class is full, prior to deletion
+            let notifyWaitlist = true;
+            const waitlistClass = await getClass(
+                parentRegistrationInput.classId,
+            );
+            if (!waitlistClass) {
+                ResponseUtil.returnBadRequest(
+                    res,
+                    `Class information could not be found`,
+                );
+                return;
+            }
+            // if class wasn't originally full, do not notify waitlist
+            if (waitlistClass._count.parentRegs < waitlistClass.spaceTotal) {
+                notifyWaitlist = false;
+            }
+
             const deletedRegistration = await deleteParentRegistration(
                 parentRegistrationInput,
             );
```

Comment:
This works but I think we can squish this into one transaction by returning the count in the `deleteParentRegistration` call and tweak the logic to say if the reg count is total space - 1, then we notify.

@aamyli  thoughts?